packages/openapi-fetch/src/index.ts

```diff
@@ -145,8 +166,8 @@ export function defaultBodySerializer<T>(body: T): string {
 }
 
 /** Construct URL string from baseUrl and handle path and query params */
-export function createFinalURL<O>(url: string, options: { baseUrl?: string; params: { query?: Record<string, unknown>; path?: Record<string, unknown> }; querySerializer: QuerySerializer<O> }): string {
-  let finalURL = `${options.baseUrl ? options.baseUrl.replace(TRAILING_SLASH_RE, "") : ""}${url as string}`;
```

Comment:
Small micro-optimization: we don’t need to be running a dynamic RegEx to remove a trailing slash. It’s much more performant to just remove a single character if needed (done above on `createClient()`, also to save cycles on running a RegEx every request